.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: c
```

Comment:
Should this be C++?

Reply:
Nope, "c" is fine. "cpp" would work, but it really doesn't matter.